redfish-core/lib/pcie.hpp

```diff
@@ -279,7 +286,8 @@ static inline void addLinkStatusToPcieDevice(
 
             std::string pcieSlotPath = path.parent_path();
 
-            getPcieSlotSubTree(asyncResp, pcieSlotPath, pcieDevice);
+            findPcieSlotServiceMap(asyncResp, pcieSlotPath, pcieDevice,
+                                   callback);
```

Comment:
I think we cast this callback to rvalue too. std::move(callback)
[Example](https://github.com/ibm-openbmc/bmcweb/blob/6b8af746da031a1a8d9cc8f90cdc2ea7ff58dd5b/redfish-core/lib/processor.hpp#L1172)